src/viam/api/app/v1/app.grpc.pb.cc

```diff
@@ -15,7 +15,7 @@
 #include <grpcpp/support/method_handler.h>
 #include <grpcpp/impl/rpc_service_method.h>
 #include <grpcpp/support/server_callback.h>
-#include <grpcpp/impl/codegen/server_callback_handlers.h>
+#include <grpcpp/impl/server_callback_handlers.h>
```

Comment:
This PR includes updates to the static proto files, which it probably shouldn't - that update should happen organically. Right before commit I will attempt to copy the state of upstream's generated sources over top of what is in this PR to bring them back in line. All changes that look to be in files renamed like `src/gen/foo -> src/viam/api/foo` should be skipped.